repos/system_upgrade/common/actors/checknfs/actor.py

```diff
@@ -34,7 +34,8 @@ def process(self):
             for mount in storage.mount:
                 if mount.tp.startswith("nfs"):
                     nfs_found = True
-                    details += "- Currently mounted NFS shares\n"
+                    details += "- Currently mounted NFS share:\n"
+                    details += "%s %s %s %s\n" % (mount.name, mount.mount, mount.tp, mount.options)
```

Comment:
```suggestion
                    details += "{} {} {} {}\n".format(mount.name, mount.mount, mount.tp, mount.options)
```

Reply:
Thinking about it, this will not cut it. If there are more options offending you will only report the first occurrence

You want to collect them and them print them out as a list. Previously it was enough to bail out on the first occurrence, but since you want to report it, you rather want all of them being reported so you can fix them all at once.

Reply:
> Thinking about it, this will not cut it. If there are more options offending you will only report the first occurrence

Oh yes.  Missed that there was a `break` on the first one.

Reply:
All NFS share occurences are now collected in lists and reported.